x-pack/legacy/plugins/maps/public/layers/styles/vector/properties/dynamic_color_property.js

```diff
@@ -111,7 +111,7 @@ export class DynamicColorProperty extends DynamicStyleProperty {
     return getColorRampStops(this._options.color);
   }
 
-  renderHeader() {
+  renderStylePropertyLegendHeader() {
```

Comment:
How about just `renderLegendHeader`? The class is already labeled as style property